WINDUP-4174: Release Notes for MTR 1.2.6 (#877)
windup/windup-documentation: docs/topics/mtr-rn-known-issues-1-2-6.adoc

// Module included in the following assemblies:
//
// * docs/release-notes-mtr/master.adoc

:_content-type: REFERENCE
[id="mtr-rn-known-issues-1-2-6_{context}"]

= Known issues

> There are no major known issues in this {ProductName} ({ProductShortName}) 1.2.6 release.

Author: ```suggestion
There are no major known issues in the {ProductShortName} 1.2.6 release.
```

> For a complete list of all known issues, see the list of link:https://issues.redhat.com/issues/?filter=12435316[MTR 1.2.6 known issues] in Jira.

Author: ```suggestion
For a complete list of all known issues, see the list of link:https://issues.redhat.com/issues/?filter=12436484[MTR 1.2.6 known issues] in Jira.
```

// Module included in the following assemblies:
//
// * docs/release-notes-mtr/master.adoc

:_content-type: REFERENCE
[id="mtr-rn-known-issues-1-2-6_{context}"]

= Known issues

The following known issues are in the {ProductShortName} 1.2.6 release:

.Unable to migrate application to {ProductShortName} due to a `SEVERE [org.jboss.windup.web.services.messaging.PackageDiscoveryMDB]` error

> When uploading files for analyze, the server log would print a `SEREVE` message, and the webpage could freeze at the discovering and fetching packages stage, returning an `SEVERE [org.jboss.windup.web.services.messaging.PackageDiscoveryMDB]` error. This error is caused by a `null: java.lang.NullPointerException`. link:https://issues.redhat.com/browse/WINDUP-4189[(WINDUP-4189)]

Author: ```suggestion
When uploading files for analyze, the server log would return a `SEVERE [org.jboss.windup.web.services.messaging.PackageDiscoveryMDB]` error. This error is caused by a `null: java.lang.NullPointerException`. link:https://issues.redhat.com/browse/WINDUP-4189[(WINDUP-4189)]
```
